packages/frontend/components/authorize/AuthorizeClaim.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+
+
+
+interface Props {
+  claim: keyof typeof claims,
+}
+
+const props = withDefaults(defineProps<Props>(), {})
+
+const claims = {
+    "identity": {
+        "header": "Identity",
+        "description": "View your basic identity info",
+    },
+    "email": {
+        "header": "Email",
+        "description": "View your email address",
+    },
+    "phone": {
+        "header": "Phone",
+        "description": "View your phone number",
+    },
+    "address": {
+        "header": "Address",
+        "description": "View your address",
+    }
+}
```

Comment:
This will be defined in the template, so it can be translated. (You can dismiss this)